test/integration_test_misc/integration_test_models_instance1/fisher.json

```diff
@@ -0,0 +1,21 @@
+{
+    "model": "fisher",
+    "storageType": "SQL",
+    "attributes": {
+      "name": "String",
+      "fisher_id": "Int"
+    },
+    "associations": {
+      "fishingRivers": {
+        "type": "many_to_many",
+        "implementation": "sql_cross_table",
```

Comment:
not sure why you are using cross table. Actually I think we have discarded this part. You can change it to 
"implementation": "foreignkeys"